From 4fe75acfbbbcc928030e805dc3eb3f1bdbda6aa7 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 30 May 2018 22:14:40 -0700
Subject: [PATCH] TRITON-380 work around logadm.conf "/var/log/*.log" entry in
 newer origins to not lose logs Reviewed by: Marsell Kukuljevic
 <marsell@joyent.com> Approved by: Marsell Kukuljevic <marsell@joyent.com>

---
 deps/sdc-scripts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/deps/sdc-scripts b/deps/sdc-scripts
index bda632a..6bfa4a2 160000
--- a/deps/sdc-scripts
+++ b/deps/sdc-scripts
@@ -1 +1 @@
-Subproject commit bda632a69ff44114f9faf689a5cd4af1fe578d66
+Subproject commit 6bfa4a298d753685c6e083d10b70b5abdc39f66f
-- 
2.21.0

